code/datums/spells/wizard.dm

```diff
@@ -333,3 +333,45 @@
 				M.Weaken(2)
 				M << "<span class='userdanger'>You're thrown back by a mystical force!</span>"
 			spawn(0) AM.throw_at(throwtarget, ((Clamp((maxthrow - (Clamp(distfromcaster - 2, 0, distfromcaster))), 3, maxthrow))), 1,user)//So stuff gets tossed around at the same time.
+
+/obj/effect/proc_holder/spell/targeted/cauterize
+	name = "Cauterize"
+	desc = "This spell will instantly heal all damage to your body. However, after a short time, half of it will dealt back to you in the form of fire damage."
+	school = "transmutation"
+	charge_max = 300
+	cooldown_min = 200 //25 deciseconds reduction per rank

```

Comment:
That's not a very charitable scale down, if a wizard wants to go full out x5 Cauterize it should be ridiculously short.